lib/runtime/wasmer/helpers.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package wasmer
+
+// #include <stdlib.h>
+//
```

Comment:
```suggestion
```

Reply:
Actually that empty line is needed for deepsource to compile successfully. Don't ask me why 😄